primeqa/pipelines/components/indexer/dense.py

```diff
@@ -107,4 +107,4 @@ def index(self, collection: Union[List[dict], str], *args, **kwargs):
             raise TypeError(
                 "ColBERT indexer expects path to `documents.tsv` as value for `collection` argument."
             )
-        self._indexer.index(self.index_name, collection)
+        self._indexer.index(self.index_name, collection, overwrite='overwrite' in kwargs and kwargs['overwrite'])
```

Comment:
@franzmpub Should we add `overwrite` with default value to datalcass properties for `IndexerComponent`? Or if not common for all indexers maybe on `ColBERTIndexer`? That way we can just take it as part of ColBERTIndexer constructor?